projects/quests.md

```diff
@@ -28,6 +30,105 @@ A quest that is intended to be completed only once would define in the payment c
 
 The evidence required for completing a quest will depend on the work expected. For a quest on writing a blog post, the evidence required will likely include a link to the blog post. It may also specify the minimum length of the blog post and that it hasn't been copied from somewhere else. For a quest on adding a feature to a web UI, the evidence required will likely include a link to the code written, some screenshots of the deployed feature and some reference to the code standard expected.
 
+## Pages
+
+### List view
+
+![](https://i.imgur.com/hRtXi66.png)
+
+This page consists of a small dashboard and a list of summarized Quests.
+
+The dasboard shows the following information:
+
+- Total of the active Quests bounties in USD
```

Comment:
maybe make some * or note about the some tokens dont show the USD value?

Reply:
Do you mean that I should list the stables here ?